index.mako

```diff
@@ -749,6 +749,9 @@ way to write a simple hook which prevents commits by file matching.  Specify
 the regex as the `entry`.  The `entry` may be any python
 [regular expression](#regular-expressions).  For case insensitive regexes you
 can apply the `(?i)` flag as the start of your entry, or use `args: [-i]`.
+The regex operates in bytes mode and is internally encoded using UTF-8 for
+that, so the hook is guaranteed to produce correct results only against files
+encoded in UTF-8 and compatible encodings.
```

Comment:
this is subtly incorrect.  for example an ascii regex will work with latin1 which is not UTF-8 compatible

Reply:
Oh, right, rephrased. It's not as concise as I'd like, but can't think of any better now.